vuestorefront/models/ir_ui_view.py

```diff
@@ -0,0 +1,42 @@
+from odoo import models
+
+from ..utils import format_vsf_cache_tags, invalidate_vsf_cache
+from .. import const
+
+
+# Should we use lower case? Because sending this to VSF cache using
+# params will make it lowercase anyway..
+TAG_PFX = "IUV"
+
+
+class IrUiView(models.Model):
+    _inherit = 'ir.ui.view'
+
+    def get_vsf_cache_tags(self):
+        return format_vsf_cache_tags(TAG_PFX, self.ids)
+
+    def write(self, vals):
+        res = super().write(vals)
+        self.invalidate_website_page_views_cache()
+        return res
+
+    def unlink(self):
+        self.invalidate_website_page_views_cache()
+        return super().unlink()
```

Comment:
How about creating an inherited view? Should it also trigger cache invalidation?

Reply:
No, you can read this in commit message: 

```
We now send request to VSF informing when website page's view have
changed. Though currently it only supports direct view changes. If
some of indirect views/templates change, it won't be triggered. For
that manual trigger was added in case it is needed to invalidate cache.
```